content/network/drivers/bridge.md

```diff
@@ -244,9 +262,15 @@ Restart Docker for the changes to take effect.
 
 ### Use IPv6 with the default bridge network
 
-If you configure Docker for IPv6 support (see [Use IPv6](#use-ipv6)), the
-default bridge network is also configured for IPv6 automatically. Unlike
-user-defined bridges, you can't selectively disable IPv6 on the default bridge.
+IPv6 can be enabled for the default bridge using options in `daemon.json`.
```

Comment:
question: would it makes sense to add a short description for the options we are setting here? I.e. that you need to specify the subnet and default gateway, this example uses example values, you need to change it to something that works for you...

Maybe redundant but as a noob I am not sure what this does, and why all of these are necessary

```suggestion
To enable IPv6 on the default bridge network, configure the following
options in the `daemon.json` configuration file:
```

Reply:
Good idea - I've added some words.